lib/scavenger_hunt/app/views/scavenger_hunt/games/_footer.html.slim

```diff
@@ -2,12 +2,12 @@
 
   = toolbar title: "Game controls" do
     = toolbar_item do
-      = link_to "End<br />Game".html_safe, finish_game_path(@game), class: "small"
+      = link_to "End<br />Game".html_safe, finish_game_path(@game), class: "small end-game"
     = toolbar_item do
       = link_to "See all<br />clues".html_safe, game_clues_path(@game), class: "small"
   h2#game-progress.sr-only Game Progress
   ul.toolbar-item.small aria-labelledby="game-progress"
-    li Total time: #{@game.total_time}
-    li
+    li.toolbar Total time: #{@game.total_time}
+    li.toolbar
```

Comment:
Adding `toolbar` style to these list items collapses all the unwanted extra space between them.